src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,16 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", "Arial", sans-serif;
+  font-weight: 500;
+}
+

```

Comment:
in reset.scss you should only reset default styles
if you want to set some default styles / indents for the page - make it via normalize.scss file